src/desktop/apps/article/components/InfiniteScrollNewsArticle.tsx

```diff
@@ -124,35 +134,57 @@ export class InfiniteScrollNewsArticle extends Component<
   }
 
   onEnter = (article, { previousPosition, currentPosition }) => {
+    const { date } = this.state
     const enteredArticle =
       previousPosition === 'above' && currentPosition === 'inside'
+    const hasNewDate = article.published_at !== date
 
-    if (enteredArticle) {
-      document.title = article.thumbnail_title
-      window.history.replaceState({}, article.id, `/news/${article.slug}`)
+    if (enteredArticle && hasNewDate) {
+      // ENTERED AN ARTICLE
+      this.setState({ date: article.published_at })
+    }
+    if (enteredArticle && article === this.props.article) {
+      this.onExpand(article)
     }
   }
 
   onLeave = (i, { previousPosition, currentPosition }) => {
-    const nextArticle = this.state.articles[i + 1]
+    const { date, articles } = this.state
+    const nextArticle = articles[i + 1]
+    const hasNewDate = nextArticle && nextArticle.published_at !== date
 
     if (
       nextArticle &&
       previousPosition === 'inside' &&
       currentPosition === 'above'
     ) {
-      document.title = nextArticle.thumbnail_title
-      window.history.replaceState(
-        {},
-        nextArticle.id,
-        `/news/${nextArticle.slug}`
-      )
+      // LEFT AN ARTICLE
+      if (hasNewDate) {
+        this.setState({ date: nextArticle.published_at })
+      }
+      document.title = 'News' // todo: replace with actual meta-title
```

Comment:
This all ties into your question about metadata on News articles - would you mind creating a Jira subtask where appropriate so we can remember to address this?

Reply:
Cool, added here: https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=GROW&modal=detail&selectedIssue=GROW-175